install.rb

```diff
@@ -294,6 +308,12 @@ def prepare_installation
     bindir = RbConfig::CONFIG['bindir']
   end
 
+  if not InstallOptions.localedir.nil?
+    localedir = InstallOptions.localedir
+  else
+    localedir = "/opt/puppetlabs/puppet/share/puppet/locale"
```

Comment:
Ahh ok... TLDR  `ProgramFiles` for binaries, `ProgramData` for config information.  This should probably be in ProgramData so you can copy-pasta to configs above and use;

`localedir = File.join(Dir::COMMON_APPDATA, "PuppetLabs", "puppet", "share", "puppet", "locale")`

@Iristyle Sound reasonable?